### PR TITLE
Update development.rst

### PR DESCRIPTION
**Pull request by LuizMarin**
Note that I did not correct the part about creating a plugin.

I suggest creating this for the cake 4.0 documentation because it is different from 2.

### CODE REVIEW

File: docs/en/development.rst [hunk 1]
```diff
@@ -188,8 +188,8 @@ templates that bake uses. The best way to do this is:
 
 #. Bake a new plugin. The name of the plugin is the bake 'theme' name
 #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
```
Comment (saeideng) [inline docs/en/development.rst]: this line forgotten

File: docs/en/development.rst [hunk 2]
```diff
@@ -188,8 +188,8 @@ templates that bake uses. The best way to do this is:
...
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/Template** to matching files in your
+#. Copy any templates you want to override from
```
Comment (saeideng) [inline docs/en/development.rst]: why extra space?

Comment (LuizMarin) [reply docs/en/development.rst]: do not know. I never used these tools

File: docs/en/development.rst (hunk 1 above)
Comment (LuizMarin) [reply docs/en/development.rst]: how would it be correct?

Comment (markstory) [reply docs/en/development.rst]: Wouldn't the path be `plugins/[name]/templates/bake/`?

File: docs/en/development.rst [hunk 3]
```diff
@@ -188,8 +188,8 @@ templates that bake uses. The best way to do this is:
 
 #. Bake a new plugin. The name of the plugin is the bake 'theme' name
 #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/Template** to matching files in your
+#. Copy any templates you want to override from 
+   **vendor/cakephp/bake/templates/bake/Template** to matching files in your
```
Comment (markstory) [inline docs/en/development.rst]:
```suggestion
   **vendor/cakephp/bake/templates/bake/** to matching files in your
```

There are more templates than just view templates.

Comment (LuizMarin) [reply docs/en/development.rst]: @markstory  

> #. Bake a new plugin. The name of the plugin is the bake 'theme' name
> #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
> 

I haven't worked with plugins yet, so I can't say anything about it

**Comment by ADmad** [comment]
Thanks for this, I have opened a new PR fixing all the paths.

**Pull request closed without merging**